problems/1.types+operators.js

```diff
@@ -2,16 +2,24 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  if (typeof val === "number") {
```

Comment:
functions looks good. You can also just return typeof val === "number" . When you have an if else and one returns true and the other returns false. Check to see if you can return just the condition itself.